File: eTeatar/eTeatar.Model/Sjediste.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Sjediste
    {
        public int SjedisteId { get; set; }

        public string Red { get; set; }

        public string Kolona { get; set; } 

        public string Status { get; set; }

        public int DvoranaId { get; set; }

        public virtual ICollection<RezervacijaSjediste> RezervacijaSjedistes { get; set; } = new List<RezervacijaSjediste>();
    }
}

File: eTeatar/UI/eteatar_desktop/lib/models/sjediste.dart
import 'package:json_annotation/json_annotation.dart';

part 'sjediste.g.dart';

@JsonSerializable()
class Sjediste {
  int? sjedisteId;
  String? red;
  String? kolona;
  String? status;
  int? dvoranaId;

  Sjediste(
    this.sjedisteId,
    this.red,
    this.kolona
  );

  factory Sjediste.fromJson(Map<String, dynamic> json) => _$SjedisteFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$SjedisteToJson(this);
}